src/main/java/com/cloudbees/jenkins/support/configfiles/TemporalFileContent.java

```diff
@@ -0,0 +1,36 @@
+package com.cloudbees.jenkins.support.configfiles;
+
+import com.cloudbees.jenkins.support.api.FileContent;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Temporal file content, auto-deleted after {@link #writeTo(OutputStream)}.
+ */
+public class TemporalFileContent extends FileContent {
```

Comment:
`TemporaryFileContent`?

I think `Temporal` has not the right meaning wrt the goal of this class.